src/pyodide/internal/workers.py

```diff
@@ -12,10 +12,62 @@
 
 import js
 
+# Get globals modules and import function from the entrypoint-helper
+from js.globalThis import pyodide_entrypoint_helper
```

Comment:
Is there any difference between this and:
```suggestion
from js import pyodide_entrypoint_helper
```

Reply:
Or even I'd suggest:
```
from js.pyodide_entrypoint_helper import doAnImport
```